nipype/interfaces/ants/utils.py

```diff
@@ -104,17 +104,21 @@ class ImageMath(ANTSCommand, CopyHeaderInterface):
     ...     op2='0.005 0.999 256').cmdline
     'ImageMath 3 structural_maths.nii TruncateImageIntensity structural.nii 0.005 0.999 256'
 
+    >>> # Test the special case of PadImage. In this case, copy_header is disabled
+    >>> # because the header cannot be copied from inputs to ouptuts.
+    >>> # When operation is PadImage, the copy_header is set to False automatically:
```

Comment:
```suggestion
    By default, Nipype copies headers from the first input image (``op1``)
    to the output image.
    For the ``PadImage`` operation, the header cannot be copied from inputs to
    outputs, and so ``copy_header`` option is automatically set to ``False``.

```

Reply:
If I remember correctly, we need Python comments so that numpydoc/napoleon conversions work well (this is partly because we are using a modified version of napoleon). I'm not 100% sure about this, but I don't think it's worth finding out.

I'll apply your suggestions, but as Python comments, if you are okay with that.

Reply:
Scratch that, looking at antsRegistration it seems we correctly handle it. I just force pushed my last commit.